### DISCUSSION TITLE
Listening first element of the Librimix dataset

### DISCUSSION THREAD

**Discussion by cemox35** [asteroid-team/asteroid#549] [Q&A]
Hi there,

I just want to listen to the first element of the librimix dataset. However, I cannot do that. Can someone help me? Thank you.

CODE:

from asteroid.data import LibriMix

train_set, val_set = LibriMix.mini_from_download(
    task='sep_clean'
)

display(Audio(train_set[0], rate=8000))

Error:
only one element tensors can be converted to Python scalars

**Comment by jonashaag** [comment]
When Googling the error message the first result is this: https://stackoverflow.com/questions/52074153/cannot-convert-list-to-array-valueerror-only-one-element-tensors-can-be-conver

The problem is that `train_set[0]` is a tensor but `Audio` expects a NumPy array. So you should use `Audio(train_set[0].numpy())` instead.
